src/schema.ts

```diff
@@ -26,5 +26,6 @@ export const typeDefs = `#graphql
   }
   type Query {
     user (id: ID!): User
+    users (limit: Int): [User]
```

Comment:
```suggestion
    users (limit: Int = 10): [User]
```

Eu tenho quase ctz que vc declarando esse 10 aqui, ele já preenche lá no resolver caso nada seja enviado 🤔

Reply:
isso mesmo!! acabei de testar e isso rola mesmo, por isso fazendo essa ateração acaba não ficando necessário a alteração de cima né? com o limit ?? 10